### PR TITLE
SKILL.tsv (updated)

### PR DESCRIPTION
**Pull request by Sourpusss**
Ran through everything again and made it so everything here should only be edits on current attributes, attributes not current in the database (in the off chance they get used later), or stuff that was discussed in the past.

### CODE REVIEW

File: EnglishTranslation-master/SKILL.tsv [hunk 1]
```diff
@@ -4582,7 +4582,7 @@ SKILL_20150714_004581	Increases the damage dealt on an enemy with [Barbed Arrow]
 SKILL_20150714_004582	Increases the damage dealt on an enemy with [Crossfire] by 1% per attribute level.
 SKILL_20150714_004583	Increases the damage dealt on an enemy with [Magic Arrow] by 1% per attribute level.
 SKILL_20150714_004584	Increases the damage dealt on an enemy with [Divine Machine Arrow] by 1% per attribute level.
-SKILL_20150714_004585	Increases the time duration of the [Bleeding] status when an enemy is hit by [Broad Head] by 2 seconds per attribute level.
+SKILL_20150714_004585	Increases the duration of [Bleeding] when an enemy is hit by [Broadhead] by 2 seconds per attribute level.
```
Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: Increases the duration of [Bleeding] by 2 seconds per attribute level when an enemy is hit by [Broadhead]. 

?

File: EnglishTranslation-master/SKILL.tsv [hunk 2]
```diff
@@ -4809,12 +4809,12 @@ SKILL_20150717_004808	Increases the AoE attack ratio of [Claymore] by 1 per attr
 SKILL_20150717_004809	When an enemy launched by [Punji Stake] hits the ground, it and nearby enemies will be dealt 50% of physical attack as damage.
 SKILL_20150717_004810	Decreases the physical defense of an enemy attacked with [Retrieve] by 10 per attribute level.
 SKILL_20150717_004811	Provides immunity to [Poison] for 2 seconds per attribute level to a character whose [Poison] has been cured due to [Detoxify].
-SKILL_20150717_004812	Even if an object gets out of the range of [Throw Gu Pot], [Poison] will continue for 2 seconds per attribute level.
+SKILL_20150717_004812	Even if an enemy steps out of [Throw Gu Pot], [Poison] will last for 2 seconds per attribute level.
```
Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: I commented on this on your first revision:

Maintains [Poison] by 2 seconds per attribute level when enemies are outside the range of [Throw Gu Pot].

or is this better?

Sustains [Poison] by 2 seconds per attribute level when enemies are outside the range of [Throw Gu Pot].

File: EnglishTranslation-master/SKILL.tsv [hunk 3]
```diff
@@ -5282,8 +5282,8 @@ SKILL_20151001_005281	Creates an explosion using 200 silver when there are no ne
 SKILL_20151001_005282	Deals 20% additional damage to Flying-type monsters when equipped with a [Two-handed Bow].
 SKILL_20151001_005283	Crossbow Mastery
 SKILL_20151001_005284	Ignores enemy defense by 5% per attribute level when equipped with a [One-handed Bow].
-SKILL_20151001_005285	Decreases the time of the first rotation by 0.3 seconds and increases the number of rotations by 0.8 for [Broom Trap] per attribute level.
-SKILL_20151001_005286	Increases the movement speed of your companion with [Praise] by 1 per attribute level.
+SKILL_20151001_005285	Decreases the time of the first rotation by 0.3 seconds and increases the number of rotations for [Broom Trap] by 0.8 per attribute level.
+SKILL_20151001_005286	Increases the movement speed of your companion under the effects of [Praise] by 1 per attribute level.
```
Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: From [Praise]?

File: EnglishTranslation-master/SKILL.tsv [hunk 4]
```diff
@@ -6010,21 +6010,21 @@ SKILL_20151224_006009	Allows you to equip the [Pistol] weapon type. You can also
...
 SKILL_20151224_006011	Allows you to equip [Leather] armor. When equipped with 3 or more pieces of [Leather] armor, increases your evasion. When equipped with 4 [Leather] armor pieces, your evasion increases from attributes by an additional 50%.
 SKILL_20151224_006012	Allows you to equip [Plate] armor.{nl}{nl}When equipped with 3 or more pieces of [Plate] armor, increases your maximum HP and physical damage taken is decreased by 5%.{nl}When equipped with 4 [Plate] armor pieces, your maximum stamina increases and physical damage taken is decreased by an additional 10%.
-SKILL_20151224_006013	Increases the companion's stamina cost cycle duration by 10 seconds per attribute level.
-SKILL_20151224_006014	When the HP of your companion is less than 50%, it will activate the [First Aid] effect and recover HP for 2 seconds. The duration is increased by 1 second per attribute level from Lv. 2.[First Aid] cannot be used for another 10 minutes after it has been activated.
+SKILL_20151224_006013	Increases the stamina reduction cycle of your companion by 10 seconds per attribute level.
```
Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: stamina -> Stamina

File: EnglishTranslation-master/SKILL.tsv (hunk 1 above)
Comment (Sourpusss) [inline EnglishTranslation-master/SKILL.tsv]: Increases the [Bleeding] duration of [Broadhead] by 2 seconds per attribute level. ?

That way it doesn't seem like the act of hitting with Broadhead extends the current bleeding stack.

Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: It says when an enemy is hit. We should include that part.

File: EnglishTranslation-master/SKILL.tsv [hunk 5]
```diff
@@ -4531,9 +4531,9 @@ SKILL_20150714_004530	Increases the damage dealt on an enemy with [Stone Shot] b
 SKILL_20150714_004531	Increases the damage dealt on an enemy with [Rapid Fire] by 1% per attribute level.
 SKILL_20150714_004532	Increases the damage dealt on an enemy with [Teardown] by 1% per attribute level.
 SKILL_20150714_004533	Increases the duration of [Scatter Caltrops] by 1 second per attribute level.
-SKILL_20150714_004534	There is a 8% chance per attribute level to  be afflicted with [Bleeding] when stepping on a caltrop.
+SKILL_20150714_004534	There is a 8% chance per attribute level that caltrops will cause [Bleeding] when stepped on.
```
Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: `8% chance per attribute level that caltrops will cause [Bleeding] when stepped on.`

File: EnglishTranslation-master/SKILL.tsv [hunk 6]
```diff
@@ -4809,12 +4809,12 @@ SKILL_20150717_004808	Increases the AoE attack ratio of [Claymore] by 1 per attr
...
+SKILL_20150717_004812	Maintains [Poison] by 2 seconds per attribute level when enemies move outside the range of [Throw Gu Pot].
 SKILL_20150717_004813	Enemies within the range of [Scan] have a chance to be afflicted with [Confusion] for 5 seconds. Increases the chance by 10% per attribute level.
 SKILL_20150717_004814	Enemies within the range of [Lachrymator] will receive damage equal to [attribute level] every 2 seconds. Increases the damage per attribute level.
 SKILL_20150717_004815	Inflicts the enemy with [Massive Bleeding] for 10 seconds when attacked with [Backstab] while affected by bleeding. Increases the duration by 1 second per attribute level. Massive Bleeding damage is proportional to the character's STR and physical attack.
 SKILL_20150717_004816	Increases the range of [Lachrymator] to 75.
-SKILL_20150717_004817	Decreases an enemy's physical defense by 1% per attribute level when you attack with [Bodkin Point].
+SKILL_20150717_004817	Further decreases an enemy's physical defense by 1% per attribute level when you attack with [Bodkin Point].
```
Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: `Further decreases an enemy's physical defense by 1% per attribute level when attacked by [Bodkin Point].`

File: EnglishTranslation-master/SKILL.tsv [hunk 7]
```diff
@@ -5854,7 +5854,7 @@ SKILL_20151224_005853	Skill Critical Rate: +#{CaptionRatio}#%
 SKILL_20151224_005854	Wild Shot
 SKILL_20151224_005855	{#DD5500}{ol}[Missile]{/}{/}{nl}Attack an enemy simultaneously with a crossbow and pistol held in both hands. 
 SKILL_20151224_005856	Sonic Strike
-SKILL_20151224_005857	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Quickly descend your hawk to attack the enemies in the targeted area.
+SKILL_20151224_005857	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Command your hawk to divebomb enemies multiple times in the targeted area.
```
Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv:5857]: `in the targeted area`

or

`in a targeted area`

?

File: EnglishTranslation-master/SKILL.tsv [hunk 8]
```diff
@@ -5891,20 +5891,20 @@ SKILL_20151224_005890	Open an offering box to receive donations in the form of i
...
-SKILL_20151224_005894	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Attack an enemy by collecting money to offer to the gods. Enemies with exception to boss monsters can be instantly killed at a 1/10 chance and the offered money can be picked back up again.
+SKILL_20151224_005894	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Attack an enemy by collecting money to offer to the gods. Enemies, with the exception of boss monsters, can be instantly killed at a 10% chance and the offered money can be picked back up again.
 SKILL_20151224_005895	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Consumes #{CaptionRatio}# silver
 SKILL_20151224_005896	Duration: #{CaptionRatio}# seconds{nl}Maximum Level: 1
-SKILL_20151224_005897	{#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Temporarily creates a magic circle that grants you a healing factor. If you or your allies take damage from an enemy inside the magic circle, HP will be quickly recovered.
+SKILL_20151224_005897	{#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Temporarily creates a magic circle that grants a healing factor. If you or your allies take damage from an enemy inside the magic circle, HP will be quickly recovered.
```
Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: `If you or your allies take damage from an enemy inside the magic circle, HP will be quickly recovered.`

or

`HP will be quickly recovered if you or your allies take damage from an enemy inside the magic circle.`

**Comment by Sunaries** [comment]
Good to merge.

**Pull request merged**